docs/creating-a-kubernetes-cluster.md

```diff
@@ -1,141 +1,54 @@
-# Creating a Kubernetes Cluster for Knative Serving
+# Creating a Kubernetes cluster
 
-Two options:
+This doc describes two options for creating a k8s cluster:
 
 * Setup a [GKE cluster](#gke)
 * Run [minikube](#minikube) locally
 
 ## GKE
 
-To use a k8s cluster running in GKE:
+1. [Install required tools and setup GCP project](https://github.com/knative/docs/blob/master/install/Knative-with-GKE.md#before-you-begin)
+   (You may find it useful to save the ID of the project in an environment variable (e.g. `PROJECT_ID`).
+1. [Create a GKE cluster for knative](https://github.com/knative/docs/blob/master/install/Knative-with-GKE.md#creating-a-kubernetes-cluster)
+1. Add `K8S_USER_OVERRIDE` to your .bashrc (see [Environment Setup](#environment-setup)):
```

Comment:
This should no longer be needed.  I think the only thing that uses it is the cluster-admin setup?